llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -390,6 +390,11 @@ static cl::opt<cl::boolOrDefault> ForceSafeDivisor(
     cl::desc(
         "Override cost based safe divisor widening for div/rem instructions"));
 
+static cl::opt<bool> UseWiderVFIfVariantsPresent(
```

Comment:
Sorry if I missed this before, but can you also rename this to something a bit more specific, i.e. `UseWiderVFIfCallVariantsPresent`? At the moment it could refer to variants of anything. Other than that the patch looks good!